composer.json

```
@@ -21,11 +21,11 @@
         }
     ],
     "require": {
-        "php": "^7.3 || ^8.0",
-        "friendsofsymfony/http-cache": "^2.15",
-        "symfony/framework-bundle": "^4.4.0 || ^5.0 || ^6.0",
-        "symfony/http-foundation": "^4.4.0 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4.0 || ^5.0 || ^6.0"
+        "php": ">= 8.1",
```

```suggestion
        "php": "^8.1",
```

i know symfony itself does this, but i strongly disagree. major PHP version changes can do BC breaks, so we should not claim compatibility with all future versions of PHP.